gourmet/timer.py

```diff
@@ -232,26 +232,34 @@ def response_cb (self, dialog, resp):
         else:
             self.close_cb()
 
-    def close_cb (self,*args):
+    def close_cb(self, *args) -> None:
+        """Close a timer window.
+
+        There are three options: either the timer is not running, and it gets
+        closed, the timer is running and it gets cancelled and closed, or the
+        timer is running, and gets hidden ready to pop back up when finished.
+        """
         self.stop_annoying()
-        msg = ("You've requested to close a window with an active timer."
-               "You can stop the timer, or you can just close the window."
-               "If you close the window, it will reappear when your timer "
-               "goes off.")
-        try:
-            do_cancel = getBoolean(label=_('Stop timer?'),
-                                   sublabel=_(msg),
-                                   custom_yes=_('Stop _timer'),
-                                   custom_no=_('_Keep timing'))
-        except UserCancelledError:
-            return  # keep the timer window open
-
-        if do_cancel or not self.timer.is_running:
+        do_cancel = True
+        if self.timer.is_running:
+            msg = ("You've requested to close a window with an active timer."
+                   "You can stop the timer, or you can just close the window."
+                   "If you close the window, it will reappear when your timer "
+                   "goes off.")
```

Comment:
There are whitespaces missing between the sentences (you can see that in the gif you uploaded). Also, this should probably go through the translation.

Reply:
Nice spotting the missing space :)
The message goes through translation below, at line 251, where `gettext` is called.